cassovary-core/src/test/scala/com/twitter/cassovary/graph/TraverserSpec.scala

```diff
@@ -248,7 +248,7 @@ class TraverserSpec extends WordSpec with MockitoSugar with Matchers {
     "yield all nodes in DFS order in expected order in unique id walk (inDir)" in {
       val dir = GraphDir.InDir
       val trav = new DepthFirstTraverser(graph, dir, Seq(10), Walk.Limits(None, None, Some(10L)))
-        with PathLengthTracker with DiscoveryAndFinishTimeTracker
+        with PathLengthTracker[Node] with DiscoveryAndFinishTimeTracker[Node]
```

Comment:
will be nice to add a test on any traverser on any subtype of Node to test the new parametrization.